Makefile

```diff
@@ -38,7 +38,17 @@ guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
 extract_translations: ## extract localizable strings from sources
-	i18n_tool extract -v
+	# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations calls this target.
+	# Otherwise, we preserve the old behavior of having segmented po files.
+	if [ "$(IS_OPENEDX_TRANSLATIONS_WORKFLOW)" = "yes" ]; then \
+		i18n_tool extract --no-segment -v; \
+		cd conf/locale/en/LC_MESSAGES; \
+		msgcat django.po wiki.po edx_proctoring_proctortrack.po mako.po underscore.po -o django_temp.po; \
+		mv django_temp.po django.po; \
+		rm wiki.po edx_proctoring_proctortrack.po mako.po underscore.po; \
```

Comment:
`underscore` should be merged into `djangojs.po`:

For demonstration purposes:

```
msgcat django.po mako.po wiki.po edx_proctoring_proctortrack.po -o django.po
msgcat djangojs.po underscore.po -o djangojs.po
```

Reply:
done